core/sdk/src/api/device/entertainment_data.rs

```diff
@@ -30,8 +30,7 @@ pub struct ContentProvider {
     pub apis: Vec<String>,
 }
 
-#[derive(Serialize, Deserialize, Debug, Clone)]
-#[cfg_attr(test, derive(PartialEq))]
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
```

Comment:
Is there any reason for including PartialEq here? I thought this is used only in the test code.